### DISCUSSION TITLE
Mongodb can not display query results

### DISCUSSION THREAD

**Discussion by stepjacky** [dbeaver/dbeaver#376] [Q&A] [answered]
on a mongodb connection
open a Mongodb Script tag window and type a Mongodb scripts like :
db.RemindEvent.find() ,then press Alt+x or click execute sql script
the result is not the record of target mongodb collection ,but is the fllowing:

false   EMBEDDED    false   EMBEDDED    local   false   EMBEDDED    false   EMBEDDED    local   RemindEvent local.RemindEvent   local.RemindEvent   #document       0.0 0.0 0.0 0.0     0.0 false

**Comment by serge-rider** [comment, marked as answer]
find() returns cursor object which has its own complex structure.
To see cursor contents try this:
`db.RemindEvent.find().toArray()`

or just
`select * from RemindEvent`
